golang.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import "context"
+
+//here is a line coment
+
+// FileGetter is usd to retriv de code.
+type FileGetter interface {
+	// GetFiles returs a FilesScanner
+	GetFiles(context.Context, *FilesRequest) (FileScanner, error)
+}
+
+func main() {
+	/*
```

Comment:
Typo inside comment in token 'lyne'. Possible corrections:line (74),lone (14),type (6)